webrtc.html

```diff
@@ -8033,17 +8035,84 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 <p>The transport over which all SCTP packets for data channels
                 will be sent and received.</p>
               </dd>
+              <dt><dfn><code>state</code></dfn> of type <span class=
+              "idlAttrType"><a>RTCSctpTransportState</a></span>, readonly</dt>
+              <dd>
+                <p>The current state of the SCTP transport. The attribute returns
+                the value of the <dfn>[[\SctpTransportState]]</dfn> internal slot.</p>
```

Comment:
Similar to my comment on the PR that added [[DtlsTransportState]]: we need to either describe when the slot's value is changed, or not use a slot.

In this case it's probably pretty simple:

1. When it's first created, the value is `connecting`? (See below comment)
2. When the four-way startup handshake finishes, queue a task to update the state to `connected`
3. When a SHUTDOWN or ABORT chunk is received, queue a task to update the state to `closed`
4. Also can go to `closed` from applying a description that rejects the data m= section, or changes the sctp-port. Or if the application closes the PeerConnection,